src/Xam.Shell.Badge.iOS/Renderers/BadgeShellItemRenderer.cs

```diff
@@ -111,7 +122,7 @@ private void ApplyBadge(int index, string text, Color bg, Color textColor)
 
                     if (badgeValue == 0)
                     {
-                        currentTabBarItem.BadgeValue = "●";
+                        currentTabBarItem.BadgeValue = IsiOS15OrNewer ? "0" : "●";
```

Comment:
Why are you changing tiny badge to 0 for iOS15+ ?  '●' seems to work just fine is you set the badge background to transparent
![image](https://user-images.githubusercontent.com/33980667/173205394-1ddb4f9f-c5a1-4dcb-a972-a2c1f0f5f7f3.png)

Reply:
I was not able to accomplish this functionality as i mentioned:

> Limitations: AFAIK individual tab colors are no longer possible see: https://developer.apple.com/forums/thread/694705

By assigning the the background color to transparent, all badges will get transparent.


There is a option to use the old functionality:

```

```